src/witan/send/vis/projection_charts.clj

```diff
@@ -0,0 +1,97 @@
+(ns witan.send.vis.projection-charts
+  (:require [clojure.core.async :as a]
+            [witan.send.chart :as wsc]
+            [witan.send.main :as wsm]
+            [witan.send.vis.general-population :as vis.gp]
+            [witan.send.vis.ingest.transitions :as vis.transitions]
+            [witan.send.vis.output-ay :as vis.oay]
+            [witan.send.vis.output-cost :as vis.ocost]
+            [witan.send.vis.output-count :as vis.ocount]
+            [witan.send.vis.output-need :as vis.on]
+            [witan.send.vis.output-setting :as vis.os]
+            [witan.send.vis.output-setting-cost :as vis.osc]))
+
+(defn charts [config-file {:keys [setting-titles-and-sets need-titles-and-sets]}]
+  (let [config                 (wsm/read-config config-file)
+        output-dir             (str (:project-dir config) "/" (get-in config [:output-parameters :output-dir]))
+        historical-transitions (vis.transitions/historical (str (:project-dir config) "/" (get-in config [:file-inputs :transitions])))
+        gen-pop-data           (into []
+                                     (comp
+                                      ;; only a subset so the charts line up
+                                      (filter #(<= 2015 (:calendar-year %)))
+                                      (filter #(< (:academic-year %) 21)))
+                                     (vis.gp/general-population (str (:project-dir config) "/" (get-in config [:file-inputs :population]))))
+        census                 (vis.transitions/->census historical-transitions)
+        settings-lookup        (into {}
+                                     (map (fn [x] [(:setting x) (:setting x)]))
+                                     census)
+        needs-lookup           (into {}
+                                     (map (fn [x] [(:setting x) (:setting x)]))
+                                     census)
+        output-ay              (future
+                                 (vis.oay/charts
+                                  (vis.transitions/ay-counts-per-calendar-year historical-transitions)
+                                  (into []
+                                        (filter #(< (:academic-year %) 21))
+                                        (vis.oay/output-ay (str output-dir "/" vis.oay/output-ay-file)))))
+        output-count           (future
+                                 (vis.ocount/chart
+                                  "2020 Baseline"
```

Comment:
This probably wants to be an argument somehow, rather than a fixed (magic) string. And I can see it elsewhere but won't add more comments